### ISSUE TITLE
Folder dropdown is not accessible

### ISSUE THREAD

**Issue by decaf-dev** [decaf-dev/obsidian-vault-explorer#162]
Just updated! Great works!

Could you please consider use text input with auto-complete rather than drop-list?

I'm using "FolderNotes" plugin and I got sooooo many folders 😹
So it's kind of hard to find a folder.
![image](https://github.com/decaf-dev/obsidian-vault-explorer/assets/4013062/8e1f8847-b3a9-4764-9a58-f033ed677c86)

An input box with suggestions of folders might be more convinient 
![image](https://github.com/decaf-dev/obsidian-vault-explorer/assets/4013062/9332b328-f9c6-438f-818a-28ec8916eff7)

_Originally posted by @Moyf in https://github.com/decaf-dev/obsidian-vault-explorer/issues/70#issuecomment-2185925447_

**Comment by decaf-dev** [comment]
@moyf Added in 1.43.0. Are there any other inputs that you would want replaced with a searchable dropdown?

**Comment by Moyf** [comment]
> @Moyf Added in 1.43.0. Are there any other inputs that you would want replaced with a searchable dropdown?

Yeah I think it's much better! Great!

( Just one more thing, could this select dropdown support "Up Arrow" and "Down Arrow" to change selection?

**Comment by decaf-dev** [comment]
Ah, yes. I will add that. I also noticed that it is not saving for the value in the settings.

**Comment by decaf-dev** [comment]
@Moyf Added up/down plus a few other fixes in 1.44.3
